src/content/translations/ro/history/index.md

```diff
@@ -140,16 +140,16 @@ Furculița Byzantium:
 - A redus recompensele [miniere](/developers/docs/consensus-mechanisms/pow/mining/) pe bloc de la 5 la 3 ETH.
 - A amânat [bomba de dificultate](/glossary/#difficulty-bomb) cu un an.
 - A adăugat capacitatea de a efectua apeluri către alte contracte care nu modifică starea.
-- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/layer-2-scaling/).
+- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/scaling/#layer-2-scaling).
```

Comment:
Similar to others, would change a;; `/layer-2-scaling/` to the main `scaling/layer-2-rollups` page:
```
/developers/docs/scaling/layer-2-rollups/
```